project/TestParallelization.scala

```diff
@@ -1,26 +1,31 @@
+import scala.util.hashing.MurmurHash3
```

Comment:
don't use this; just use scala.util.random.nextInt for a more even distribution

Reply:
can we use a fixed seed to make it deterministic? otherwise it might be hard to track stuff when trying to optimize further.

Reply:
Upon reflection, I'm actually satisfied with `scala.util.hashing.MurmurHash3`. It's deterministic and has better even distribution properties than normal hashcode